### PR TITLE
Adds bindings field to stack

### PR DESCRIPTION
**Pull request by mrajashree**
`Bindings` field of type `map[json]` is added to `stack.json`. 
On `/v2-beta/projects/1a5/stacks`, on clicking the `Create` button its value can be set with other keys of stack.
It's value should be of the form:
`services: {"scale":2,"ports":[],"labels":{}}`
Tests are pending

### CODE REVIEW

**Comment by mrajashree** [comment]
@ibuildthecloud I've added the `bindings` field to Stack and it is visible on /v2-beta.
I haven't added tests since tests for v2 are pending

**Comment by ibuildthecloud** [comment]
@mrajashree Thanks, once I get the v2 tests commited you can rebase and add tests.

**Comment by mrajashree** [comment]
@ibuildthecloud The build for v1 passed but for v2 failed. The error is an assertion error in a test in test_authorization.py, but my changes aren't related to that. So I don't understand why exactly the build failed

File: resources/content/schema/base/bindingsDef.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"resourceFields": {
+		"services": {
+			"type": "servicesDef"
```
Comment (ibuildthecloud) [inline resources/content/schema/base/bindingsDef.json]: Do you mean `map[servicesDef]`?

File: tests/integration/cattletest/core/test_authorization.py [hunk 2]
```diff
@@ -192,9 +192,13 @@ def test_user_types(user_client, adds=set(), removes=set()):
         'nfsConfig',
         'blkioDeviceOption',
         'scalePolicy',
+        'bindings',
+        'serviceBinding',
     }
     types.update(adds)
     types.difference_update(removes)
+    print types
```
Comment (ibuildthecloud) [inline tests/integration/cattletest/core/test_authorization.py]: Please remove this print

**Comment by mrajashree** [comment]
@ibuildthecloud two tests are failing in the build and I don't think related to my changes

**Comment by deniseschannon** [comment]
https://github.com/rancher/rancher/issues/3732

**Pull request merged**